core/trino-main/src/test/java/io/trino/operator/output/BenchmarkPartitionedOutputOperator.java

```diff
@@ -222,34 +312,105 @@ private DriverContext createDriverContext(TaskContext taskContext)
                     .addDriverContext();
         }
 
-        private TaskContext createTaskContext()
+        private TaskContext createTaskContext(Session session)
```

Comment:
you don't need to pass `session` as an argument (it's already a fiend in context)